From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Luis Padron <lpadron@squareup.com>
Date: Fri, 19 Jan 2024 15:17:39 -0500
Subject: Patch testonly swift_library


diff --git a/BUILD.bazel b/BUILD.bazel
index 0ae5406..42e81b5 100644
--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -25,5 +25,6 @@ swift_library(
     deps = [
         ":iOSSnapshotTestCaseCore"
     ],
+    testonly = True,
     visibility = ["//visibility:public"]
 )
-- 
2.42.1

